app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -40,10 +40,10 @@ def validate_and_decode_token
       end
 
       def get_jwt_identity
-        # Will be changed when real get identity implemented
         @logger.debug(LogMessages::Authentication::AuthnJwt::GET_JWT_IDENTITY.new)
         jwt_identity
         @logger.debug(LogMessages::Authentication::AuthnJwt::FOUND_JWT_IDENTITY.new(jwt_identity))
+        @jwt_identity_initialized = true
```

Comment:
WDYT instead of using this bool to initialize the `@jwt_identity` at the start of this class with decided const on an "empty user/ not identified/ not initilized "

and before writing the audit check if its value was updated to a real value

Reply:
@tzheleznyak also can you check if we have this flow in the test plan of failing before identified the user  and to check the audit message ?

Reply:
Talked about this. Moved to const so we will be able to configure the empty username later.
I will stay with the boolean